### DISCUSSION TITLE
card height when in landscape mode

### DISCUSSION THREAD

**Discussion by Vansmak** [punxaphil/custom-sonos-card#122] [General]
I am trying to run this with the player in panel mode and it is well off screen when i have the tablet in landscape orientation.  I have tried card-mod height and max-height but it doesnt help.  Hopefully someone can help me here. Thank you.

**Comment by punxaphil** [comment]
First time I hear about this problem. What is your screen size?
Please also share some screenshots.

**Reply by Vansmak** [reply]
I will later but it not just this card, ant Fullscreen card, or panel mode, trun your screen to landscape and it overflows the screen
